src/components/marginals/Footer/Footer.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import footer from '../../../../config/content/Footer';
```

Comment:
Would suggest using absolute instead of relative imports :)

Reply:
done. [commit link](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/a3cb7fccba0cdf4019181b48074cb7b64d280606)